src/components/search/discoveryArea.tsx

```diff
@@ -57,11 +70,21 @@ export default function DiscoveryArea({ schema }): JSX.Element {
   }
   return (
     <Grid container>
-        <Grid className="w-full px-[1em] sm:px-[2em] sm:mt-32 max-md:max-w-full shadow-none aspect-ratio bg-lightviolet">
-          <Grid container className="container mx-auto pt-[2em] sm:pt-0">
-            <SearchArea schema={schema} header="Data Discovery" />
-          </Grid>
+      <Grid className="w-full px-[1em] sm:px-[2em] sm:mt-32 max-md:max-w-full shadow-none aspect-ratio bg-lightviolet">
+        <Grid container className="container mx-auto pt-[2em] sm:pt-0">
+          <SearchArea schema={schema} header="Data Discovery" />
         </Grid>
+      </Grid>
+
+      <Grid className="w-full px-[1em] sm:px-[2em] max-md:max-w-full shadow-none aspect-ratio bg-lightbisque">
+        <Grid container className="container mx-auto py-[1em] px-4">
+          <Banner>
+            This platform is under development, feel free to
+            <BannerLink href={'#'} target={'_blank'}>share your feedback &rarr;</BannerLink>
```

Comment:
Should this URL be parameterized? 🤔 
```suggestion
            <BannerLink href={'https://go.illinois.edu/DATA-DISCOVERY-FEEDBACK'} target={'_blank'}>share your feedback &rarr;</BannerLink>
```